Chapters/01-intro.Rmd

```diff
@@ -34,7 +36,7 @@ where $\hat{y}_t$ is the point forecast. This equation does not explain where th
 
 In addition, we will discuss in this textbook two types of models:
 
-1. Additive, where the components are added to some other components;
+1. Additive, where all components are added to each other;
```

Comment:
Not sure about this. When all components are added, then this is a "pure additive" model. I would prefer a more loose sentence here, in order to avoid questions about mixed models. e.g. the components of mu can be additive, but they can be multiplied by error term. And I don't want to go into many details about what are the components and how they can interact with each other.

Reply:
"where most components are added to one another"